src/main/app/src/app/zaken/zaken.service.ts

```diff
@@ -180,6 +180,12 @@ export class ZakenService {
         );
     }
 
+    deleteBetrokkene(uuid: string): Observable<Zaak> {
```

Comment:
wat voor uuid is dit? Een rolUuid? een betrokkeneUuid? Naam even uitbreiden

Reply:
Pas ik aan.